tests/kafkatest/services/kafka/kafka.py

```diff
@@ -301,6 +301,23 @@ def alter_message_format(self, topic, msg_format_version, node=None):
         self.logger.info("Running alter message format command...\n%s" % cmd)
         node.account.ssh(cmd)
 
+    def configure_throttle(self, topic, throttle, node_count):
+        node = self.nodes[0]
+
+        self.logger.info("Throttling all replicas on topic %s with limit %s", topic, throttle)
+        cmd = "%s --zookeeper %s --entity-name %s --entity-type topics --alter --add-config quota.replication.throttled.replicas=*" % \
+              (self.path.script("kafka-configs.sh", node), self.zk.connect_setting(), topic)
+        self.logger.info("Running throttled replicas command...\n%s" % cmd)
+        node.account.ssh(cmd)
+
+        for brokerId in range (0, node_count):
+            self.logger.info("Setting throttle limit for broker %s with limit %s", str(brokerId), throttle)
+            cmd = "%s --zookeeper %s --entity-name %s --entity-type brokers --alter --add-config replication.quota.throttled.rate=%s" % \
+                  (self.path.script("kafka-configs.sh", node), self.zk.connect_setting(), str(brokerId), str(throttle))
```

Comment:
how does this differ from the pervious `kafka-configs` command on line 308? It seems one sets the throttle for a topic, and the other sets the throttle for the broker as a whole. Is this true? If so, shouldn't these be set separately? If not, then why do we need two invocations?